CONTRIBUTORS.md

```diff
@@ -1,6 +1,6 @@
 
```

Comment:
Changes to contributors should be made using the bot, please remove these files from the PR.